pyocd/coresight/cortex_m.py

```diff
@@ -766,6 +800,10 @@ def write_core_registers_raw(self, reg_list, data_list):
                 specialRegValue = self.read_core_register(CORE_REGISTER['cfbp'])
                 break
 
+            if (reg >= 0x10000) and (reg <= 0x10007):
+                specialRegValue = self.read_core_register(CORE_REGISTER['xpsr'])
```

Comment:
Just realised these two aren't exclusive for a list - need to be remembering xpsr and cfbp separately.